### PR TITLE
fix(image): fix build errors for the features

### PR DESCRIPTION
**Pull request by kwonoj**
### Description

Update missing fixes for the features.

Closes PACK-2722

### CODE REVIEW

File: crates/turbopack-image/src/process/mod.rs [hunk 1]
```diff
@@ -268,13 +268,10 @@ fn encode_image(image: DynamicImage, format: ImageFormat, quality: u8) -> Result
...
         ImageFormat::WebP => {
-            use image::codecs::webp::{WebPEncoder, WebPQuality};
-            WebPEncoder::new_with_quality(&mut buf, WebPQuality::lossy(quality)).write_image(
-                image.as_bytes(),
-                width,
-                height,
-                image.color(),
-            )?;
+            use image::codecs::webp::WebPEncoder;
+            let encoder = WebPEncoder::new_lossless(&mut buf);
```
Comment (ForsakenHarmony) [inline crates/turbopack-image/src/process/mod.rs:272]: Do we want it to be lossless? Probably not?

Comment (kwonoj) [reply crates/turbopack-image/src/process/mod.rs:272]: doesn't seem to other ctor even, if I'm reading correctly https://github.com/image-rs/image/blob/6edf8ae492c4bb1dacb41da88681ea74dab1bab3/src/codecs/webp/encoder.rs#L19

Comment (ForsakenHarmony) [reply crates/turbopack-image/src/process/mod.rs:272]: that's unfortunate, maybe create a linear ticket to look into some way to get lossy webp back?

Comment (kwonoj) [reply crates/turbopack-image/src/process/mod.rs:272]: yeah will do

**Pull request merged**